### ISSUE TITLE
Accessibility - Download section of exhibitions/events

### ISSUE THREAD

**Issue by filipranik** [bratislava/gmb.sk#274]
Add format, size and strapi title for aria-label.

**Why we want to change it**
The current implementation does not meet accessibility requirements

**Current design:**
The screen reader reads the original file name, which may not be understandable.
![Image](https://github.com/user-attachments/assets/80c1a6dc-b48e-4e1a-8bb1-6b9c3df59277)
